### DISCUSSION TITLE
wikipedia page view for POI prioritization

### DISCUSSION THREAD

**Discussion by j5k** [organicmaps/organicmaps#1446] [Ideas]
As an idea input: the [castle map](https://castle-map.infs.ch) is using the wikipedia page view for prioritization of POIs. Maybe a cool feature since many places, mountains, etc. have a wikidata tag.

**Comment by patepelo** [comment]
We are now discussing this on the open issue #6332 if you would want to participate and give some ideas. It is indeed a very good proposition. Thank you.

**Reply by j5k** [reply]
Thank you for the hint
